js/currencycom.js

```diff
@@ -461,7 +464,7 @@ module.exports = class currencycom extends Exchange {
             const margin = swap; // as we decided to set
             if (swap) {
                 symbol = symbol.replace (this.options['leverage_markets_suffix'], '');
-                symbol += ':' + quote;
+                symbol += ':' + 'QUANTO';
```

Comment:
@ttodua from that thread, I got the idea that `quanto` would be the swap type but the actual symbol would be something like:
```
BTC/USD:GBP
BTC/USD:EUR
(...)
```

Reply:
> @ttodua from that thread, I got the idea that `quanto` would be the swap type but the actual symbol would be something like:
> 
> ```
> BTC/USD:GBP
> BTC/USD:EUR
> (...)
> ```

that is one possible way, but I wouldn't vote for that approach. It would leave a false impression that they are (as it seems to say alike all other ccxt exchanges) separate markets, however they are not. So, the only way (which I see as the only correct way atm from my view) is to have `BTC/USD:QUANTO` and while opening an order for that market, you will need to add `settleCurrency`:
```
createOrder ('BTC/USD:QANTO', ..., ..., ..., {settleCurrency: 'EUR'})
```
On first sight you might think that 'noobs' find it harder to find out, but I still think that it's quite much better that we had such 'a bit harder' approach, however, all users would be able to use it without any further confusion and complications. Thus, the market will be only one (one chart, one orderbook, shortly- **one market**) where the only difference is to be able to open an order for user in different settle currency (which affect only that user). so, `settleCurrency` would be best place

Reply:
@ttodua yeah but let's say you want to `fetchTickers`, how would you specify two different Quanto markets?

Reply:
> @ttodua yeah but let's say you want to `fetchTickers`, how would you specify two different Quanto markets?

can you please say an example of what do you mean? i.e. for QUANTO markets, user need to use `BTC/USDT:QUANTO` as there is no ` BTC/USDT:USDT` and `BTC/USDT:EUR` or whatever markets.  per my understanding, the unbreaking rule of CCXT should be that "one symbol = one market".  So, users should just know (like they know our unified future symbol formats) that they should refer/handle quanto markets with `:QUANTO` suffix. This wont be breaking change as none of "current ccxt format" is being changed. this is just a new approach that we had discussed https://discord.com/channels/690203284119617602/945747867853783101/972203014037319720 and i think this might be the best approach. however, please lmk what you think (if otherwise)

Reply:
The market has its precision (which is being set atm) https://github.com/ccxt/ccxt/pull/14769/files#diff-7b15bf0e3c33deddbffc32893fcb98de6f03fde868463ee0b48884c9fefba32bR480 and also limits.
the settlecurrency might need `costToPrecision` and that's all imo.

ofc, this is just an open PR, whatever you @lead decide, be it so. lmk then.